annotation-processor/src/main/java/com/bloxbean/cardano/client/plutus/annotation/processor/blueprint/DataTypeProcessUtil.java

```diff
@@ -207,6 +208,8 @@ public FieldSpec processPlutusDataType(String javaDoc, BlueprintSchema schema, S
     private String getPackage(String ns) {
         String pkg = (ns != null && !ns.isEmpty())? annotation.packageName() + "." + ns + ".model"
                 : annotation.packageName() + ".model";
+
+        pkg = pkg.toLowerCase();
```

Comment:
wondering if we should also remove `-` and `_` ?

Reply:
Thanks @nemo83 . Done.